R/utility.R

```diff
@@ -1,35 +1,46 @@
-getInstallationCommand <- function(packages){
-  installation <- ""
-
-  for(package in packages){
-    installation <- paste0(installation,
-                         sprintf("Rscript -e \'args <- commandArgs(TRUE)\' -e \'install.packages(args[1], dependencies=TRUE)\' %s", package),
-                         ";")
+getJobPackageInstallationCommand <- function(type, packages){
+  script <- ""
+  if (type == "cran") {
+    script <- "Rscript $AZ_BATCH_JOB_PREP_WORKING_DIR/install_cran.R"
+  }
+  else if (type == "github") {
```

Comment:
No, we are creating a command line and the script handles the packages. 

Rscript $AZ_BATCH_JOB_PREP_WORKING_DIR/install_cran.R httr ludridate hts